client/plots/profilePlot.js

```diff
@@ -205,6 +204,7 @@ export class profilePlot {
 				type: 'radio',
 				chartType,
 				settingsKey: 'isAggregate',
+				styles: { display: 'inline-block' },
```

Comment:
The option is passed here @xzhou82 if this is what you meant, so it looks like this:

<img width="1043" alt="Screenshot 2024-10-15 at 1 24 49 PM" src="https://github.com/user-attachments/assets/a4063780-dac0-43b3-9507-ab6d93ad3f98">